### PR TITLE
Update to Alpine 3.18, copy Dockerfile for 2.8.2-beta.2 pre-release

### PR DESCRIPTION
**Pull request by tianon**
https://github.com/distribution/distribution/releases/tag/v2.8.2-beta.2 :partying_face:

### CODE REVIEW

File: Dockerfile.rc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM alpine:3.18
```
Comment (tianon) [inline Dockerfile.rc:1]: Because GitHub struggles to realize this is a copy, here's a raw diff between the two `Dockerfile`s:

```diff
$ diff -u Dockerfile*
--- Dockerfile	2023-05-09 16:40:01.367003754 -0700
+++ Dockerfile.rc	2023-05-09 16:41:53.699869027 -0700
@@ -4,15 +4,15 @@
 
 RUN set -eux; \
 # https://github.com/distribution/distribution/releases
-	version='2.8.1'; \
+	version='2.8.2-beta.2'; \
 	apkArch="$(apk --print-arch)"; \
 	case "$apkArch" in \
-		x86_64)  arch='amd64';   sha256='f1a376964912a5fd7d588107ebe5185da77803244e15476d483c945959347ee2' ;; \
-		aarch64) arch='arm64';   sha256='4c588c8e62c9a84f1eecfba4c842fe363b91be87fd42e3b5dac45148a2f46c52' ;; \
-		armhf)   arch='armv6';   sha256='d711b3b6e77f3acc7c89fad9310413ef145751ac702627dc1fa89991bf3b6104' ;; \
-		armv7)   arch='armv7';   sha256='d2f2180c1a847056f9c5dcfd1d6fda4c6086d126204541e0ed047c04f30a0f91' ;; \
-		ppc64le) arch='ppc64le'; sha256='ca77cdfb7b1415869558c118b5e783bb9d695a74d8426a0bd8d8a39beb23fb85' ;; \
-		s390x)   arch='s390x';   sha256='3e505af15c562870869441d5d1f79988c3c666b9a4370894ecf2f064860b48ba' ;; \
+		x86_64)  arch='amd64';   sha256='f16957fe46a797439892bcea255c9e98477b47a56e03df4906e920eb2db57c28' ;; \
+		aarch64) arch='arm64';   sha256='f1c713711bf547185b82139bb4413b0409a6849be918dc29ea08ec04a6f72806' ;; \
+		armhf)   arch='armv6';   sha256='364d4da480737627849892e4656bccdda4370007fac655b02e0c3cebe87e14a6' ;; \
+		armv7)   arch='armv7';   sha256='e989455bd142736873fbd77ea6e574a079958644527d8716e381fc6545a91a8a' ;; \
+		ppc64le) arch='ppc64le'; sha256='377f0f56d4a9eab32630856a6357b245d89050137e433b538c35aafc3b713580' ;; \
+		s390x)   arch='s390x';   sha256='66f63e84d7adf02d0b7d8a821c618c464970de3d1726dd1a13c2149a71798f53' ;; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
 	wget -O registry.tar.gz "https://github.com/distribution/distribution/releases/download/v${version}/registry_${version}_linux_${arch}.tar.gz"; \
```

**Comment by thaJeztah** [comment]
oh man you beat me by 1 minute! 😂

**Comment by milosgajdos** [comment]
Wild speed! 🚀

**Comment by tianon** [comment]
It would've been faster if you hadn't published the release while I finally stopped for a second for a bio break!! :joy:

**Review by thaJeztah — COMMENTED** [review]
LGTM as well!

mine didn't update alpine to 3.18, so only diff;

```patch
git diff --no-index Dockerfile.rc Dockerfile.seb
diff --git a/Dockerfile.rc b/Dockerfile.seb
index 04e8cc3..d515771 100644
--- a/Dockerfile.rc
+++ b/Dockerfile.seb
@@ -1,4 +1,4 @@
-FROM alpine:3.18
+FROM alpine:3.16

 RUN apk add --no-cache ca-certificates
```

**Comment by thaJeztah** [comment]
FWIW; We're calling it a day, and will do a non-beta tomorrow (but will be same code, only version change, and a `.toml` file added for the release notes.

**Pull request merged**